project-codeflare/codeflare-sdk
# build: update python to v3.9 and ray to v2.7

# Issue link


closes https://github.com/project-codeflare/codeflare-sdk/issues/371

# What changes have been made

Update python and ray versions

# Verification steps



1. Checkout PR
2. Build python wheel using `poetry build`. Wheel should be in the `dist` folder.
3. Install the wheel on your cluster
4. run through the demo notebooks
5. the nbs should run as expected

## Checks
- Testing Strategy
